lips/lip-2.md

```diff
@@ -0,0 +1,1430 @@
+---
+lip: 2
+title: Libra Roles and Permissions
+author: Tim Zakian, Sam Blackshear
+status: Draft
+type: Informational
+created: 06/26/2020
+---
+
+# Summary
+---
+This LIP describes the conceptual model and implementation of access control on the Libra blockchain.
+
+---
+# Abstract / Motivation
+---
+
+Libra uses a variant of role-based access control (RBAC) to restrict access to sensitive on-chain operations.
+
+A *role* is an entity with some authority in the Libra system. The roles in the initial system are:
+LibraRoot - the root authority of Libra. Controlled jointly by Libra Payment Networks and the Association Council 
```

Comment:
Please add formatting (specifically add bullets here), line breaks disappear in md